.github/workflows/ci.yml

```
@@ -0,0 +1,84 @@
+name: CI
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '* * * * 0'
+jobs:
+  checks:
+    name: Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - 1.40.0
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: sudo apt -y install libudev-dev
```

This will probably have to be updated to match #67:

https://github.com/parasyte/pixels/pull/67/files#diff-354f30a63fb0907d4ad57269548329e3R17

```suggestion
        run: sudo apt -y install libsdl2-dev
```

---

Done :)